### ISSUE TITLE
Loading error when user checks in late evening 

### ISSUE THREAD

**Issue by gdkoo** [gdkoo/SleepWakeRegulate#10]
sunsetRoundHrEpoch not found in sunsetHrArray
- Found when checking in EST 10PM (sunset RoundHrEpoch displays sunset for following day, whereas sunsetHrArray is still for the current day)

**Comment by gdkoo** [comment]
#13

**Comment by gdkoo** [comment]
Unsure if this is the same error -> but there are sporadic results when checking sunset at 12AM. Seems to execute fine.. but it shouldn't.

**Comment by gdkoo** [comment]
**Description**

When checking after 8 PM, instead of showing a message telling the user to go outside, the display shows loading. 

**Expected Behavior:**

- eveningMainDisplay executes
- eveningMessage is defined
- eveningMessage is displayed
- timer shows max and min times

******What’s Happening:******

- eveningMessage is undefined
- timer doesn’t load because of error

**************Error Message:**************

```jsx
main.8f62a86b86b54720.js:8000 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'eveningMessage')
    at _callee$ (main.8f62a86b86b54720.js:8000:33)
    at tryCatch (main.8f62a86b86b54720.js:7979:1357)
    at Generator.<anonymous> (main.8f62a86b86b54720.js:7979:4174)
    at Generator.next (main.8f62a86b86b54720.js:7979:2208)
    at asyncGeneratorStep (main.8f62a86b86b54720.js:7980:103)
    at _next (main.8f62a86b86b54720.js:7981:194)
```

************************Reproduce************************

- check app between 8-10 PM in a given timezone

************Files:************

src/calculation/epochWeatherMethods.js

src/components/eveningMainDisplay.js

******************Tasks:******************

- [ ]  Debug error message in console
- [ ]  Identify what the time that epoch hour from epochHourlyMethods and the sunsetRoundtoHour returns and if they align

**Comment by gdkoo** [comment]
Resolved Task #1: If-else statements in <components/index.js> were not in the correct sequence and tooLateDisplay.js would not execute. The correct order resolved the issue such that:
- eveningMessage is defined
- timer shows expected behavior
